install/installer/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/jetstack/cert-manager v1.4.4
 	github.com/replicatedhq/kots v1.67.0
 	github.com/spf13/cobra v1.2.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
It does not appear to have anything to do with this PR 🤔

Reply:
correct, just a side effect of running `leeway run components:update-go-modules`